SummaryStats/app.py

```diff
@@ -0,0 +1,46 @@
+import os
+import logging
+import flask
+from flask import Flask, request
+import xarray as xr
+import rioxarray as rxr
+import geopandas as gpd
+from glob import glob
+
+logging.basicConfig()
+logging.root.setLevel(logging.INFO)
+
+app = Flask(__name__)
+
+
+def summary_stats(gdf: gpd.GeoDataFrame, ds: xr.DataArray | xr.Dataset):
```

Comment:
Looks like a little clean up need here

Reply:
Summary Stats is next up.  Yes this is messy at the moment